src/classes/Commands.ts

```diff
@@ -1227,6 +1230,140 @@ export = class Commands {
+                return;
+            } else {
+                const item = SKU.fromString(sku);
+                const name = this.bot.schema.getName(item, false);
+
+                this.bot.tf2gc.deleteItem(params.assetid, err => {
+                    if (err) {
+                        log.warn(`Error trying to delete ${name}: `, err);
+                        this.bot.sendMessage(steamID, `Failed to delete ${name}(${params.assetid}): ${err.message}`);
+                        return;
+                    }
+                    this.bot.sendMessage(steamID, `Deleted ${name}(${params.assetid})!`);
+                });
+                return;
+            }
+        }
+
+        if (params.name !== undefined || params.item !== undefined) {
+            this.bot.sendMessage(
+                steamID,
+                `Please only use sku property.
+
+                Below are some common items to delete:
+                • Smissamas Sweater: 16391;15;untradable;w1;pk391
+                • Soul Gargoyle: 5826;6;uncraftable;untradable
+                • Noice Maker - TF Birthday: 536;6;untradable
+                • Bronze Dueling Badge: 242;6;untradable
+                • Silver Dueling Badge: 243;6;untradable
+                • Gold Dueling Badge: 244;6;untradable
+                • Platinum Dueling Badge: 245;6;untradable
+                • Mercenary: 166;6;untradable
+                • Soldier of Fortune: 165;6;untradable
+                • Grizzled Veteran: 164;6;untradable
+                • Primeval Warrior: 170;6;untradable
+                • Professor Speks: 343;6;untradable
+                • Mann Co. Cap: 261;6;untradable
+                • Mann Co. Online Cap: 994;6;untradable
+                • Proof of Purchase: 471;6;untradable
+                • Mildly Disturbing Halloween Mask: 115;6;untradable
+                • Seal Mask: 582;6;untradable
+                • Pyrovision Goggles: 743;6;untradable
+                • Giftapult: 5083;6;untradable
+                • Spirit Of Giving: 655;11;untradable
+                • Party Hat: 537;6;untradable
+                • Name Tag: 5020;6;untradable
+                • Description Tag: 5044;6;untradable
+                • Ghastly Gibus: 584;6;untradable
+                • Ghastlier Gibus: 279;6;untradable
+                • Power Up Canteen: 489;6;untradable
+                • Bombinomicon: 583;6;untradable
+                • Skull Island Topper: 941;6;untradable
+                • Spellbook Page: 8935;6;untradable
+                • Gun Mettle Campaign Coin: 5809;6;untradable
+                • MONOCULUS!: 581;6;untradable
+                
+                Or other items, please refer here: https://bit.ly/3gZQxFQ (defindex)`
+            );
+            return;
+        }
+
+        if (params.sku === undefined) {
+            this.bot.sendMessage(steamID, 'Missing item sku');
+            return;
+        }
+
+        const uncraft = params.sku.includes(';uncraftable');
```

Comment:
Shouldn't you use the variable you set on Line 132 to get `item.craftable` ? This is better than checking the direct string from the SKU.